src/main/java/com/easyrest/dao/OrderDao.java

```diff
@@ -0,0 +1,94 @@
+        int insertedOrderId = 0;
+        try (Connection connection = DBManager.openConnection()) {
+            int restaurantId = getRestaurantId(connection, email);
+            try (PreparedStatement addOrderStatement = connection.prepareStatement(INSERT_WAITING_FOR_CONFIRM_ORDER_QUERY)) {
+                addOrderStatement.setInt(1, restaurantId);
+                ResultSet addOrderResultSet = addOrderStatement.executeQuery();
+                while (addOrderResultSet.next()) {
+                    insertedOrderId = addOrderResultSet.getInt(1);
+                }
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return insertedOrderId;
+    }
+
+    public int saveAcceptedOrder(String email) {
+        int insertedOrderId = 0;
+        try (Connection connection = DBManager.openConnection()) {
+            int restaurantId = getRestaurantId(connection, email);
+            try (PreparedStatement addOrderStatement = connection.prepareStatement(INSERT_ACCEPTED_ORDER_QUERY)) {
+                addOrderStatement.setInt(1, restaurantId);
+                ResultSet addOrderResultSet = addOrderStatement.executeQuery();
+                while (addOrderResultSet.next()) {
+                    insertedOrderId = addOrderResultSet.getInt(1);
+                }
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return insertedOrderId;
+    }
+
+    public void deleteOrder(int id) {
+        try (Connection connection = DBManager.openConnection();
+             PreparedStatement deleteOrderStatement = connection.prepareStatement(DELETE_ORDER_QUERY)) {
+            deleteOrderStatement.setInt(1, id);
+            deleteOrderStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private int getRestaurantId(Connection connection, String email) {
+        int administratorId = 0;
+        int restaurantId = 0;
+        try (PreparedStatement administratorIdStatement = connection.prepareStatement(USER_ID_BY_EMAIL_QUERY)) {
+            administratorIdStatement.setString(1, email);
+            ResultSet administratorResultSet = administratorIdStatement.executeQuery();
+            while (administratorResultSet.next()) {
+                administratorId = administratorResultSet.getInt("id");
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+        try (PreparedStatement restaurantIdStatement = connection.prepareStatement(RESTAURANT_ID_BY_ADMINISTRATOR_ID_QUERY)) {
+            restaurantIdStatement.setInt(1, administratorId);
+            ResultSet restaurantResultSet = restaurantIdStatement.executeQuery();
+            while (restaurantResultSet.next()) {
+                restaurantId = restaurantResultSet.getInt("id");
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
```

Comment:
why did you throw another type of exception?

Reply:
Fixed, removed catch block and add method throws.